src/app/log/[id]/Log.controller.tsx

```diff
@@ -3,48 +3,77 @@
 import useLogModel from "./Log.model";
 import LogView from "./Log.view";
 import useMasilMapStore from "@/components/MasilMap/store/useMasilMapStore";
-import { MASILS_DATA, USER_DUMMY_DATA } from "./Log.constants";
-import { GeoPosition } from "@/types/OriginDataType";
 import { TabType } from "./Log.types";
 
 interface LogControllerProps {
-  baseLocation: GeoPosition;
+  logId: string;
 }
 
-const LogController = ({ baseLocation }: LogControllerProps) => {
-  const { tabIndex, setTabIndex, currentPinIndex, setCurrentPinIndex, mapCenter, setMapCenter } =
-    useLogModel({ baseLocation });
+const LogController = ({ logId }: LogControllerProps) => {
+  const {
+    masilData,
+    tabIndex,
+    setTabIndex,
+    currentPinIndex,
+    setCurrentPinIndex,
+    mapCenter,
+    setMapCenter,
+    baseLocation,
+    userInfo,
+  } = useLogModel({ logId });
   const { setIsOutCenter } = useMasilMapStore();
 
-  const handlePinIndex = (PinIndex: number) => {
+  if (!masilData) {
+    return;
+  }
+
+  const handleCurrentPinIndex = (PinIndex: number) => {
     setCurrentPinIndex(PinIndex);
     setTabIndex(TabType.Pin);
 
-    if (MASILS_DATA.pins.length === 0) {
+    if (masilData.pins.length === 0) {
       return;
     }
 
-    const { lat, lng } = MASILS_DATA.pins[PinIndex].point;
+    const { lat, lng } = masilData.pins[PinIndex].point;
     setMapCenter({ lat, lng });

```

Comment:
[ P5 ]
고냥 요렇게 하면 또 짧아질 수 있을것같아서 한번 올려봤슴당

```suggestion
    const { point } = masilData.pins[PinIndex];
    setMapCenter(point);
```